frontend/src/components/DatabaseSchemaReview/components/SchemaReviewCard.vue

```diff
@@ -71,9 +77,10 @@ import { PropType } from "vue";
 import { useRouter } from "vue-router";
 import { DatabaseSchemaReviewPolicy } from "../../../types/schemaSystem";
 import { useEnvironmentStore } from "@/store";
+import { environmentName } from "../../../utils";
```

Comment:
We can use `@/utils` instead of relative path `../../../utils`.

Reply:
Updated